public/ng-views/maps.html

```diff
@@ -394,36 +395,100 @@ <h6 ng-repeat="(key, variant) in variants" ng-show="variant.sel">
 </leaflet>
 <leaflet  center   = "center_right"
           defaults = "defaults_right"
-          maxbounds = "maxBounds_right"
           id       = "map_right"
           tiles    = "tiles_right"
           >
 </leaflet>
 
 
-<div class="basecol right" ng-show="overview" style="max-height: 80%">
+<!--CONTROL PANEL ROUTE SELECTOR MODULE--> 
```

Comment:
This whole portion of the code is very dangerous - dangerous in the sense that this sort of code is not correctly assembled. CSS stands for cascading style sheets - HTML should be assembled accordingly. There needs to be a super div that all this lies in, and then nested within that are these components, similar to how I had each side a super div, and then within each side (left or right), there is a bottom portion and a top portion. Thus, everything was segmented into quadrants.